workers/ohsome_quality_analyst/api/response_models.py

```diff
@@ -22,12 +23,29 @@ class TopicResponse(ResponseBase):
 
 
 class TopicListResponse(ResponseBase):
```

Comment:
Shall we keep "List" in the class name or shall we rename it to something like: `TopicsResponse` or `AllTopicsResponse`?

Reply:
fixed in 4303eba0